### DISCUSSION TITLE
Not able to download payment invoice.

### DISCUSSION THREAD

**Discussion by bhupal501** [community/community#59937] [New to GitHub]
Hi Team,
I invested on a subscription to Github Teams. I must provide the invoice to our financial department. However, I cannot download the invoice. Help me out here, please.

**Comment by queenofcorgis** [comment]
Hi there, 

Would you mind elaborating on not being able to download? Do you see a button to do so, you are more likely to get a useful response if you are explicit about your situation, giving a few more details might help someone give you a nudge in the right direction.
